src/App.svelte

```diff
@@ -115,6 +132,31 @@
     scrollBottom = !scrollBottom;
     editor.focus();
   }
+  async function rawGithubGetRequest(url) {
+    const res = await fetch(url, {
+        method: 'GET'
+    });
+    
+    if (!res.ok)
+        throw new Error(await res.text());
+
+    return res;
+}
+async function rawGithubTextRequest(url) {
+    const res = await rawGithubGetRequest(url);
+    return await res.text();
+}
+
+async function rawGithubJSONRequest(url) {
+    const res = await rawGithubGetRequest(url);
+    return await res.json();
+}
+export async function loadGuide() {
```

Comment:
no export

Reply:
ah yeah i missed taking this out once I ditched the modal approach, took it out now